databases/week3/meal_application.sql

```diff
@@ -0,0 +1,33 @@
+CREATE DATABASE meal_application;
+USE meal_application;
+
+CREATE TABLE `Meal` (
+  `id` int,
```

Comment:
`id` INT NOT NULL AUTO_INCREMENT PRIMARY KEY,

Reply:
Fixed. Thank you